app.rb

```diff
@@ -0,0 +1,66 @@
+require 'sinatra/base'
```

Comment:
Although this is generally well laid out, I'd suggest you take a look at RESTful design for your routes. The review document for the challenge gives a specific example of how to structure your routes in a RESTful way, and there are tonnes of good resources online. By applying them, you'll end up with a more consistent route naming policy that also consistently utilises the GET/POST/DELETE stuff too